Generic_Plugin_Admin.php

```diff
@@ -235,6 +238,20 @@ function admin_enqueue_scripts() {
 		wp_register_script( 'w3tc-widget', plugins_url( 'pub/js/widget.js', W3TC_FILE ), array(), W3TC_VERSION );
 		wp_register_script( 'w3tc-jquery-masonry', plugins_url( 'pub/js/jquery.masonry.min.js', W3TC_FILE ), array( 'jquery' ), W3TC_VERSION );
 
+		// New feature count for the Feature Showcase.
+		wp_register_script( 'w3tc-feature-counter', plugins_url( 'pub/js/feature-counter.js', W3TC_FILE ), array(), W3TC_VERSION, true );
```

Comment:
@cssjoe its better to avoid pushing module-specific stuff into generic code.
breaks modularity.

Reply:
I can move pub/js/feature-counter.js to Generaic_AdminFeatureShowcase.js in a patch release.